src/Kliva/ViewModels/SidePaneViewModel.cs

```diff
@@ -174,5 +173,13 @@ internal void ShowHide(Type pageType = null)
                 this.IsPaneOpen = false;
             }
         }
+
+        private void ChangePage<DestinationPageType>()
+        {
+            if (typeof(DestinationPageType) != _pageType)
```

Comment:
This will break navigating from someone else's profile page to your own profile page, as it's the same page type